workflow/controller/controller.go

```diff
@@ -408,13 +411,19 @@ func (wfc *WorkflowController) processNextPodCleanupItem() bool {
 		pods := wfc.kubeclientset.CoreV1().Pods(namespace)
 		switch action {
 		case labelPodCompleted:
-			_, err := pods.Patch(podName, types.MergePatchType, []byte(`{"metadata": {"labels": {"workflows.argoproj.io/completed": "true"}}}`))
+			_, err := pods.Patch(
+				ctx,
+				podName,
+				types.MergePatchType,
+				[]byte(`{"metadata": {"labels": {"workflows.argoproj.io/completed": "true"}}}`),
+				metav1.PatchOptions{},
+			)
```

Comment:
Is this formating from a linter or did you do this yourself? Just wondering

Reply:
I don't remember. I think I did it manually to make it easier to read given that the arguments list is pretty long